src/clients/ProfitClient.ts

```diff
@@ -63,6 +64,11 @@ export class ProfitClient {
   }
 
   isFillProfitable(deposit: Deposit, fillAmount: BigNumber) {
+    if (toBN(deposit.relayerFeePct).lt(toBNWei(this.minRelayerFeePct))) {
+      this.logger.debug({ at: "ProfitClient", message: "Relayer fee % < 0.03%" });
```

Comment:
You should change this to say
```suggestion
      this.logger.debug({ at: "ProfitClient", message: "Relayer fee % < minimum relayer fee %",  minRelayerFeePct: this. minRelayerFeePct });
```